### DISCUSSION TITLE
v0.31.1

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#2052] [General]
In this version:
* Make WDL_mu score type work as intended.
* Fix macos CI builds.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.31.1'>v0.31.1</a>.</em>

**Comment by tonyl12x12** [comment]
What is the difference between LC0 v0.31.0 and v0.31.1? Can I use NN T4 1024 V.Large network with an RTX 3080 and 1TB SSD or should I use T3 768 network at say game in 40mins + 8s inc?

**Reply by borg323** [reply]
The only functional change is the score display.

**Comment by ghost** [comment]
GPU-ONNX-DML does not work on my laptop. 

Processor	AMD Ryzen 5 5625U with Radeon Graphics            2.30 GHz
Installed RAM	8.00 GB (7.34 GB usable)
Device ID	89883396-5F95-4306-8C76-CD0DD89ECE47
Product ID	00356-24583-63310-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 11 Home Single Language
Version	23H2
Installed on	‎10/‎9/‎2022
OS build	22631.4317
Experience	Windows Feature Experience Pack 1000.22700.1041.0

**Reply by borg323** [reply]
Did you run the install script? If yes, there is not enough information for an answer, you can ask in the #help channel in our [discord chat](https://discord.gg/pKujYxD) for guidance.

**Reply by ghost** [reply]
@borg323 okay thanks. It is working now. I just clicked install.cmd.
